src/extension/index.ts

```diff
@@ -184,13 +200,13 @@ function activateVirtualDocuments(disposables: Disposable[], store: Store) {
             if (contents?.rendered?.text) return contents?.rendered?.text;
             if (contents?.text) return contents?.text;
             if (contents?.binary) {
-                const lines = Buffer.from(contents?.binary, 'base64').toString('hex').match(/.{1,32}/g) ?? [];
-                return lines.reduce((sum, line, i) => {
+                const lines = Buffer.from(contents?.binary, 'base64').toString('hex').match(/.{1,32}/g);
+                return lines?.reduce((sum, line, i) => {
```

Comment:
My local TypeScript version was throwing red errors on this line. The changes aren't required, but I figure they at some point will be when this repo's TS is upgraded. Up to you if you want me to revert the changes, I'm fine either way.

Reply:
What was the TS error saying? I'm not seeing the same.

Reply:
```
This expression is not callable.
  Each member of the union type '{ (callbackfn: (previousValue: string, currentValue: string, currentIndex: number, array: string[]) => string): string; (callbackfn: (previousValue: string, currentValue: string, currentIndex: number, array: string[]) => string, initialValue: string): string; <U>(callbackfn: (previousValue: U, currentValue: string, ...' has signatures, but none of those signatures are compatible with each other.ts(2349)
```
![image](https://user-images.githubusercontent.com/2766036/208736022-7c246b9b-c017-4366-aa45-ef42b3521eb8.png)

Which, after looking it up, seems to be because lines has a type of `RegExpMatchArray | []`, but `.reduce` has to act on a specific type of array.

Reply:
That's a good reason. Let's keep your change.